src/test/resources/county-court-judgment/by-instalments.json

```diff
@@ -0,0 +1,29 @@
+{
+  "defendant": {
+    "type": "individual",
+    "title": "Dr.",
+    "name": "John Smith",
+    "address": {
+      "line1": "52",
+      "line2": "Down Street",
+      "city": "Manchester",
+      "postcode": "DF1 3LJ"
+    },
+    "serviceAddress": {
```

Comment:
we don't have a serviceAddress

Reply:
removed. 

serviceAddress = null in SampleTheirDetails and other tests updated.